### PR TITLE
feat(ext/fetch): Response.json()

### LINKED ISSUE #13902: Implement `Response.json()`
https://github.com/whatwg/fetch/pull/1392 adds support for a static Response.json method to the Fetch spec.

WPTs have already been written, and are available at https://github.com/web-platform-tests/wpt/pull/32825.

### PR DESCRIPTION
**Pull request by AaronO**
A simple/tentative implementation of https://github.com/whatwg/fetch/issues/1389

Closes #13902

### CODE REVIEW

File: ext/fetch/23_response.js [hunk 1]
```diff
@@ -209,6 +210,18 @@
...
 
+    /**
+     * @param {any} value
+     * @param {ResponseInit} init
+     * @returns {Response}
+     */
+    static json(value, init = {}) {
+      const body = JSONStringify(value);
+      const response = new Response(body, init);
+      response.headers.set("Content-Type", "application/json");
```
Comment (andreubotella) [inline ext/fetch/23_response.js:221]: The `Response` constructor when passed a string, `ArrayBuffer`, `FormData`, etc. will only set the `Content-Type` header if one wasn't passed in the `init` object. `Response.json()` should probably do the same.

**Review by lucacasonato — CHANGES_REQUESTED** [review]
Pending WPT and spec changes. Do not merge.

File: ext/fetch/23_response.js (hunk 1 above)
Comment (AaronO) [reply ext/fetch/23_response.js]: Handled by 2142d7b61

**Comment by lucacasonato** [comment]
Thanks @AaronO, but closing in favor of #14566. I implemented the change as per the latest spec revision there, and enabled the WPTs.

**Pull request closed without merging**